sdks/v4-sdk/src/multicall.ts

```diff
@@ -9,12 +9,14 @@ export abstract class Multicall {
    */
   private constructor() {}
 
-  public static encodeMulticall(calldatas: string | string[]): string {
-    if (!Array.isArray(calldatas)) {
-      calldatas = [calldatas]
+  public static encodeMulticall(calldataList: string | string[]): string {
+    if (!Array.isArray(calldataList)) {
+      calldataList = [calldataList]
     }
 
-    return calldatas.length === 1 ? calldatas[0] : Multicall.INTERFACE.encodeFunctionData('multicall', [calldatas])
+    return calldataList.length === 1
+      ? calldataList[0]
+      : Multicall.INTERFACE.encodeFunctionData('multicall', [calldataList])
```

Comment:
Since `calldataList` is already an array, do we need to wrap it again in an array?

Reply:
because the mutlicall signature requires [], and the underlying functions being called in periphery require a list of actions (calldataList)